### ISSUE TITLE
[BUG] security vulnerabilities

### ISSUE THREAD

**Issue by pharindoko** [getfider/fider#1112]
**Self Hosted**
fider 21.1

**Describe the bug**
Found a lot of security vulnerabilities scanning the current stable image
https://trivy.dev/results/?image=getfider/fider:stable

**To Reproduce**
1. Go to 'https://trivy.dev/'
2. Click on 'Scan'

**Expected behavior**
Regular dependency updates for container images

**Comment by goenning** [comment]
Hey Florian, can you try running on getfider/fider:main?

This one is 2 months old, so it should report less vulnerabilities , but I'll build another image soon to catch the latest patches from upstream images

**Comment by goenning** [comment]
I tried re-scanning the image (`getfider/fider:main`), but it seems to be using a cached version. Do you know how to force a pull and re-scan it?

**Comment by pharindoko** [comment]
Hey @goenning 

I guess your best option is to scan your local docker image using trivy.

**Comment by pharindoko** [comment]
> Hey Florian, can you try running on getfider/fider:main?
> 
> This one is 2 months old, so it should report less vulnerabilities , but I'll build another image soon to catch the latest patches from upstream images

Yes the main versions has less critical issues found.
But I would like to use the last stable state.

**Comment by goenning** [comment]
Yeah, it's been a while since the last release, I'll get a new one released soon.

**Comment by asos-robbell** [comment]
Hi @goenning — any update on this? Bumping to the following will resolve all of the critical vulnerabilities:

```
FROM --platform=${TARGETPLATFORM:-linux/amd64} golang:1.19-bookworm AS server-builder 
...
FROM --platform=${TARGETPLATFORM:-linux/amd64} node:16-bookworm AS ui-builder 
...
FROM --platform=${TARGETPLATFORM:-linux/amd64} debian:bookworm-slim
```

Happy to raise a PR

**Comment by goenning** [comment]
Hey @asos-robbell I'd be happy to review and merge a PR. Thanks
